aliroemhildt/whats-cookin
# Add reset file, fixed accessibility issues

#### What's this PR do?
-adds the reset.scss file
-changes heading succession issues we had -> this brought our accessibility score to 100
-made some updates to css to make more user friendly/responsive

#### Where should the reviewer start?
-reset.scss file - we did not write this but found and applied to project
-index.scss, index.html, and _variables.scss have some alterations to increase accessibility and general userability

#### How should this be manually tested?
-go to site, play around, notice the responses
-do a lighthouse audit- 100% baby!


#### Any background context you want to provide?
-we aren't currently, huge fans of our table -> it's shifted a bit since we imported the reset file; will circle back

#### What are the relevant tickets?


#### Screenshots (if appropriate)


#### Questions:
